bucket/conan.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "1.15.1",
+    "description": "The open-source, decentralized C/C++ package manager.",
+    "homepage": "https://conan.io/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/conan-io/conan/releases/download/1.15.1/conan-win-64_1_15_1.exe",
+            "hash": "sha1:592a541943d4715389a070ff9584ba627016bd02"
+        },
+        "32bit": {
+            "url": "https://github.com/conan-io/conan/releases/download/1.15.1/conan-win-32_1_15_1.exe",
+            "hash": "sha1:5b8d570ede92fb4e54b37394bc885f1b5a94cf6b"
+        }
+    },
+    "innosetup": true,
+    "bin": [
+        "conan\\conan.exe",
+        "conan\\conan_build_info.exe",
+        "conan\\conan_server.exe"
+    ],
+    "checkver": {
+        "github": "https://github.com/conan-io/conan/"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/conan-io/conan/releases/download/$version/conan-win-64_$underscoreVersion.exe"
+            },
+            "32bit": {
+                "url": "https://github.com/conan-io/conan/releases/download/$version/conan-win-32_$underscoreVersion.exe"
+            }
+        },
+        "hash": {
+            "url": "https://dl.bintray.com/conan/installers/$basename.sha1"
```

```suggestion
            "url": "https://dl.bintray.com/conan/installers/$basename.sha256"
```

SHA256 is better than SHA1 in security.